packages/renderer/rollup.config.js

```diff
@@ -39,20 +38,23 @@ const getExternal = ({ browser }) => [
   /@babel\/runtime/,
   'react/jsx-runtime',
   ...(browser ? [] : ['fs', 'path', 'url']),
-  ...Object.keys(pkg.dependencies).filter(
-    (name) => name !== 'react-reconciler',
-  ),
+  ...Object.keys(pkg.dependencies),
+  // ...Object.keys(pkg.dependencies).filter(
+  //   (name) => name !== 'react-reconciler',
+  // ),
   ...Object.keys(pkg.peerDependencies),
 ];
 
 const getPlugins = ({ browser, declarationDests, minify = false }) => [
   json(),
   ...(browser ? [ignore(['fs', 'path', 'url'])] : []),
-  alias({
-    entries: {
-      'react-reconciler': 'react-reconciler/cjs/react-reconciler.production.js',
```

Comment:
Not really sure why this was needed, not bundling `react-reconciler` seems like a better way forward?